### PR TITLE
utk: add a help entry in the visitor struct, ListCLI,  and a help command

### PR DESCRIPTION
**Pull request by rminnich**
Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: cmds/utk/utk.go [hunk 1]
```diff
@@ -69,6 +71,12 @@ func main() {
 		log.Fatal("at least one argument is required")
 	}
 
+	// help can't be registered as a visitor because it has nothing to visit.
+	if flag.NArg() == 1 && flag.Args()[0] == "help" {
```
Comment (rjoleary) [inline cmds/utk/utk.go]: Instead, can you put this in the `flag.Usage`? Like:

```
flag.Usage = func() {
   ...
}
```

**Comment by rjoleary** [comment]
Thanks! This fixes #148

File: pkg/visitors/extract.go [hunk 2]
```diff
@@ -141,7 +141,7 @@ func (v *Extract) Visit(f uefi.Firmware) error {
 
 func init() {
 	var fileIndex uint64
-	RegisterCLI("extract", 1, func(args []string) (uefi.Visitor, error) {
+	RegisterCLI("extract", "extract the files to a directory", 1, func(args []string) (uefi.Visitor, error) {
```
Comment (GanShun) [inline pkg/visitors/extract.go]: You seem to have inconsistent capitalization on the beginning of the help string. should we standardize on lower case for everything?

**Comment by GanShun** [comment]
Also, this is going to conflict with #155, so you'll need to fix it up when ryan merges 155

**Comment by rminnich** [comment]
I'll wait but sure I can add it to usage

**Comment by rjoleary** [comment]
#155 is now merged, you'll need to add help string to `count`.

File: pkg/visitors/extract.go (hunk 2 above)
Comment (rminnich) [reply pkg/visitors/extract.go]: yes

**Pull request merged**